apps/src/templates/PaneHeader.jsx

```diff
@@ -192,10 +192,24 @@ export const PaneButton = Radium(function(props) {
     }
   }
 
+  function onKeyDownWrapper(event) {
+    const {onClick} = props;
+    if (
+      event.key === ' ' ||
```

Comment:
This is new to me - what key does this represent?

Reply:
Oof. To be completely honest, I'm not sure either 😬. I copied it straight from here: https://benfrain.com/converting-divs-into-accessible-pseudo-buttons/

Reply:
Oh huh. Looks like it's the newer version of "SpaceBar" 
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values

Reply:
Oh interesting! Today I learned!